test/Microsoft.Azure.WebJobs.Host.EndToEndTests/WebJobs.Host.EndToEndTests.csproj

```diff
@@ -36,6 +36,15 @@
     <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
     <LangVersion>default</LangVersion>
   </PropertyGroup>
+  <PropertyGroup>
+    <SignAssembly>true</SignAssembly>
+  </PropertyGroup>
+  <PropertyGroup>
+    <AssemblyOriginatorKeyFile>..\..\src\Common\PublicKey.snk</AssemblyOriginatorKeyFile>
```

Comment:
I had it public at first (I think you saw the getter/setter comment) but realized that no one should ever really be setting this unless they want to run two hosts on the same machine, which I figured would only be us in the tests. 

But this does open up the possibility of running two hosts locally for testing, which you'd asked about in another comment.

The risk is that someone doesn't understand this and manually sets the HostMachineId and then deploys to two instances -- they'd basically be exchanging the lease back and forth.

But I can add a comment that the setter should only be used for running multiple hosts on the same machine...